### ISSUE TITLE
New firmware update has broken plugin!

### ISSUE THREAD

**Issue by coltonshipley** [emmcc/node-switchmate#8]
A new firmware dropped for the original switchmates. The new firmware is 2.99.99 and it seems that node-switchmate no longer works properly. I can't find any devices with the ./discover.switchmate command. I've ran with sudo and without. 

This also breaks compatibility with homebridge.

Hopefully this is an easy fix and can be fixed soon.

Thanks!

**Comment by armaanbindra** [comment]
Yes, same problem here, switchmate was part of my whole smart home setup to control some of the lights in addition to many other things, but this firmware update broke everything. 

![bitmoji](https://render.bitstrips.com/v2/cpanel/7197569-171899267_20-s1-v1.png?transparent=1&palette=1&width=246)
My Version Number and Details:
<img src="https://user-images.githubusercontent.com/6989720/31420062-df16698c-ae04-11e7-9d85-565936f06819.png" alt="alt text" width="250" >

When the old version was working I had an awesome setup with my own raspberry pi server that controls many other devices etc. simultaneously, and that controlled the switchmate switch via bluetooth, and I could communicate to the raspberry pi via my google home and IFTT, so It was as simple as "Ok google, lights on" and everything was good to go. So I would love to get this up and running as soon as possible, and will try to help as much as I can myself. Also the new app claims to support google assistant but even that is broken :(

**Comment by psolin** [comment]
I'm on 2.99.15 for Switchmate firmware / Mac OS X 10.12.6, and I can confirm that discovery is non-responsive.  Just bought it -- never synced before.  I hope that this is repaired sometime soon.

**Comment by brianpeiris** [comment]
I've written up some details about the new firmware in my repo, if anyone wants to dive deeper into fixing this:
https://github.com/brianpeiris/switchmate/issues/6#issuecomment-338525059

**Comment by valkjsaaa** [comment]
I figured how to downgrade the Switchmate firmware. It should be helpful for people who have this issue. Check here: [switchmate-firmwares](https://github.com/valkjsaaa/switchmate-firmwares/).
